src/components/document/canvas.tsx

```diff
@@ -344,9 +344,14 @@ export class CanvasComponent extends BaseComponent<IProps, IState> {
 
   private getDocumentToShow = () => {
     const {showPlaybackControls, historyDocumentCopy: documentToShow} = this.state;
-    if (showPlaybackControls && documentToShow) {
+    if (showPlaybackControls && documentToShow && documentToShow.key === this.props.document?.key) {
       return documentToShow;
     } else {
+      // If we are showing a new document, make sure the history document gets updated. This may not happen
+      // otherwise if the user is using the document scroller and the playback controls are open.
+      if (showPlaybackControls) {
+        this.handleTogglePlaybackControlComponent();
+      }
```

Comment:
These changes handle the situation where a user has opened the history playback controls for a document, and then opens a different document using the document scroller. Previously, if the user tried to open a new document with the playback controls present, the previous document would remain in view.